nomad/job_endpoint_test.go

```diff
@@ -363,8 +363,11 @@ func TestJobEndpoint_Register_ACL(t *testing.T) {
 		t.Run(tt.Name, func(t *testing.T) {
 			codec := rpcClient(t, s1)
 			req := &structs.JobRegisterRequest{
-				Job:          tt.Job,
-				WriteRequest: structs.WriteRequest{Region: "global"},
+				Job: tt.Job,
+				WriteRequest: structs.WriteRequest{
+					Region:    "global",
+					Namespace: tt.Job.Namespace,
```

Comment:
Hopefully with the above fix these test fixes aren't necessary.

Reply:
Given that the number of tests is small and sets a path not being used in live-systems, I think the explicitness outweigh the tests benefit.

Reply:
I've decided to leave tests explicit.  Less implicit parts in tests the better imo.  Also, follows pattern of setting region in these tests despite it having default.